doc/sphinx-guides/source/user/dataset-management.rst

```diff
@@ -143,8 +143,12 @@ DVUploader is a community-developed tool, and its creation was primarily support
 File Handling
 =============
 
-Certain file types in Dataverse are supported by additional functionality, which can include downloading in different formats, file-level metadata preservation, file-level data citation; and exploration through data visualization and analysis. See the sections below for information about special functionality for specific file types.
+Certain file types in Dataverse are supported by additional functionality, which can include downloading in different formats, previews, file-level metadata preservation, file-level data citation; and exploration through data visualization and analysis. See the sections below for information about special functionality for specific file types.
 
+File Previews
+-------------
+
+Installations of Dataverse can install previewers for common file types uploaded by their research communities. The previews appear on the file page. If a preview tool for a specific file type is available, the preview will be created and will display automatically. File previews are not available for restricted files. 
```

Comment:
I just checked with @sekmiller and "File previews are not available for restricted files" is not entirely accurate. If you are logged into to Dataverse and have access to a restricted file and there is no guestbook and no terms of use, you will see the preview. That's my understanding anyway. It's the same or similar logic as when we show "Public URL" for files.

Reply:
Nice, I thought it was simple logic based on the file's status. I'll update this.

Reply:
https://github.com/IQSS/dataverse/commit/a992e0efe0ab817bc0afec5b9bf355868440e72a